packages/addons/src/plugins/autoHeightTextarea.ts

```diff
@@ -12,16 +12,26 @@ export function createAutoHeightTextareaPlugin(): FormKitPlugin {
   const autoHeightTextareaPlugin = (node: FormKitNode) => {
     if (node.props.type !== 'textarea') return
     node.addProps(['autoHeight'])
+    node.addProps(['maxAutoHeight'])
 
     node.on('created', () => {
       const autoHeight = undefine(node.props.autoHeight)
+      const maxAutoHeight = node.props.maxAutoHeight
```

Comment:
It would be nice if someone could use string-based numbers too, for example `<FormKit type="textarea" max-auto-height="500" />`".

```suggestion
      const maxAutoHeight = Number.isFinite(node.props.maxAutoHeight) ? parseInt(node.props.maxAutoHeight) :  undefined
```